src/components/UI/Form/Input/Input.tsx

```diff
@@ -120,6 +122,7 @@ export const Input: React.SFC<InputProps> = ({ textArea = false, disabled = fals
             {helperText}
           </FormHelperText>
         ) : null}
+        {dropdown && dropdown}
```

Comment:
```suggestion
        {dropdown}
```

If its null then it would not show.